README.md

```diff
@@ -449,6 +449,7 @@ _[SIP](https://en.wikipedia.org/wiki/Session_Initiation_Protocol)/[IPBX](https:/
 - [flaskbb](https://flaskbb.org/) - FlaskBB is forum software written in Python using the microframework Flask. You can easily create new topics, posts and send other users private messages. It also includes basic administration and moderation tools. ([Source Code](https://github.com/sh4nks/flaskbb)) `BSD-3-Clause` `Python`
 - [FluxBB](http://fluxbb.org/) - Fast, light, user-friendly forum software for your website. ([Source Code](https://github.com/fluxbb/fluxbb)) `GPL-2.0` `PHP`
 - [Friendica](https://friendi.ca/) - Social Communication Server. ([Source Code](https://github.com/friendica/friendica)) `AGPL-3.0` `PHP`
+- [Glosa](https://github.com/glosa/glosa-server) - Open source commentary system easy to integrate with static pages. You can import from Disqus. ([Source Code](https://github.com/glosa/glosa-server)) `GPL-3.0` `Clojure`
```

Comment:
@tanrax `Java` is more appropriate because it needs the JRE to run and users/admins who will instal this will care about requirements more than they care about what exact language it was written in. Please update the language/platform to `Java`.

Reply:
I think we should change that rule asap. It is pointless writing Java there cause the repo contains 0 java soruce files. Otherwise we would need to add C to everything cause most languages are written in C.

Reply:
Also Typescript would wrong everywhere and replacing it with javascript is as equally wrong cause you can compile it to wasm.

Reply:
@nodiscc I'm finding 3 software that use the Clojure language but don't mention that they use Java: Metabase, Accelerated Text y Todomini. I am happy to make the change, but I just wanted to inform you that you already have others with this formula :)

Reply:
> I think we should change that rule asap. It is pointless writing Java there cause the repo contains 0 java soruce files. Otherwise we would need to add C to everything cause most languages are written in C.

>Also writing Docker to languages is as equal wrong cause the software is never written in Docker language and as a sysadmin I care about what language is used behind inside the docker container when I need to debug stuff and fix bugs.

@SuperSandro2000 please open an issue to discuss this, and I will explain why this reasoning is wrong :) this PR is not the place to do it

> I'm finding 3 software that use the Clojure language but don't mention that they use Java: Metabase, Accelerated Text and Todomini.

We should review these, indeed. Let's wait for @SuperSandro2000's issue.

> I am happy to make the change, but I just wanted to inform you that you already have others with this formula

Yep, please change to `Java`

Reply:
@nodiscc update :smile:

Reply:
I opened an issue to discuss this further. #2110 I hope I summarized the points correctly.